client/my-sites/plan-features-2023-grid/components/billing-timeframe.tsx

```diff
@@ -8,7 +8,7 @@ import {
 	TERM_ANNUALLY,
 } from '@automattic/calypso-products';
 import { formatCurrency } from '@automattic/format-currency';
-import { localize, TranslateResult, useTranslate } from 'i18n-calypso';
+import i18n, { localize, TranslateResult, useTranslate } from 'i18n-calypso';
```

Comment:
Any reason to use the default import here, instead of important `getLocaleSlug` directly ? 🤔

Reply:
Ah! I quickly skimmed the index.js exports for i18n-calypso but didn't see `getLocaleSlug`, so I just imported `i18n` instead. Using the named export is better, so good catch 🙂

Reply:
[05e1fb6](https://github.com/Automattic/wp-calypso/pull/76738/commits/05e1fb66025dbd90b3a77ddc90f2c8a62e7dfcdb)